src/seedu/addressbook/ui/Printable.java

```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.ui;
+
+public interface Printable {
+	
+	String getPrintableString(Printable... printables);
```

Comment:
please read the description carefully; the method inside the interface doesn't accept parameters.